assets/content/less/buttons/buttons.less

```diff
@@ -0,0 +1,73 @@
+// lesshint qualifyingElement: { enabled: true, allowWithClass: true, allowWithAttribute: true }
+.btn {
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-size: 14px;
+    padding-left: 29px;
+    padding-right: 31px;
+
+    &:hover {
+        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.23);
+    }
+
+    // Blue button
+    &.btn-primary,
+    &.btn-primary:hover {
+        background: #65a6de;
+        border: 1px solid #5a93c4;
+        border-radius: 3px;
+        color: #ffffff;
+    }
+
+    // Green button
+    &.btn-success,
+    &.btn-success:hover {
+        background: #80c235;
+        border: 1px solid #72ab31;
+        border-radius: 3px;
```

Comment:
You could Not duplicate this rule across all buttons and just define it once for all of them